data/remnant ships.txt

```diff
@@ -437,6 +515,63 @@ ship "Pelican"
 
 
 
+ship "Petrel"
+	sprite "ship/petrel"
+	thumbnail "thumbnail/petrel"
+	attributes
+		category "Fighter"
+		licenses
+			Remnant
+		"cost" 1141000
+		"shields" 2300
+		"hull" 1000
+		"cargo space" 20
+		"fuel capacity" 100
+		"ramscoop" 1
+		"required crew" 2
+		"bunks" 3
+		"mass" 50
+		"drag" 1.4
+		"heat dissipation" .9
+		"outfit space" 165
+		"weapon capacity" 49
+		"engine capacity" 34
+		"shield generation" 0.85
+		"shield energy" 0.55
+		"hull repair rate" 0.38
+		"hull energy" 0.28
+		"gaslining" 1
+		"outfit scan power" 12
+		"outfit scan speed" 1
+		"tactical scan power" 26
+		"asteroid scan power" 30
+		weapon
+			"blast radius" 15
+			"shield damage" 150
+			"hull damage" 200
+			"hit force" 350
+	outfits
+		"Inhibitor Cannon" 3
+		
+		"Millennium Cell" 2
+		"Crystal Capacitor"
+		"Thermoelectric Cooler"
+		
+		"Crucible-Class Thruster"
+		"Crucible-Class Steering"
+		
+	gun 0 -38.5 "Inhibitor Cannon"
+	gun -10.5 -24 "Inhibitor Cannon"
+	gun 10.5 -24 "Inhibitor Cannon"
+	turret 0 2.5
+	engine 0 41.5
+	explode "tiny explosion" 30
+	explode "small explosion" 20
+	description "The Petrel is a multi-role mining/wing leader fightercraft designed to provide support for flights of lighter Terns in high risk environments. The ship's design originated as an a urban mobile gun platform that flitted around structures and natural terrain, but proved easy to adapt into a spacecraft."
```

Comment:
```suggestion
	description "The Petrel is a multi-role mining/wing leader fighter craft designed to provide support for flights of lighter Terns in high risk environments. The ship's design originated as an a urban mobile gun platform that flitted around structures and natural terrain, but proved easy to adapt into a spacecraft."
```
i suppose both is valid but i felt inclined to mention it

Reply:
Thanks for pointing it out. If both are valid, I think I will keep it as-is.